Libraries/pbxbuild/Sources/Target/Environment.cpp

```diff
@@ -282,59 +276,67 @@ Create(Build::Environment const &buildEnvironment, Build::Context const &buildCo
     pbxproj::XC::BuildConfiguration::shared_ptr targetConfiguration;
     ext::optional<pbxsetting::XC::Config> projectConfigurationFile;
     ext::optional<pbxsetting::XC::Config> targetConfigurationFile;
+
     {
-        // FIXME(grp): $(SRCROOT) must be set in order to find the xcconfig, but we need the xcconfig to know $(SDKROOT). So this can't
-        // use the default level order, because $(SRCROOT) comes below $(SDKROOT). Hack around this for now with a synthetic environment.
-        // It's also in the wrong order because project settings should be below the SDK, but are needed to *load* the xcconfig.
+        /*
+         * Create a synthetic build setting environment to determine the SDK to use. The real build
```

Comment:
Yeah. I found out a while ago that it was expected behavior, but I forgot to update this comment.